5calls/app/src/main/java/org/a5calls/android/a5calls/util/AnalyticsManager.kt

```diff
@@ -6,7 +6,9 @@ import org.a5calls.android.a5calls.BuildConfig
 import org.a5calls.android.a5calls.model.AccountManager
 
 class AnalyticsManager {
-    private val staticProps: Map<String, String> = mapOf("isAndroidApp" to "true")
+    private val staticProps: Map<String, String> = mapOf(
+        "isAndroidApp" to "true",
+        "version" to BuildConfig.VERSION_CODE.toString())
```

Comment:
can we call this `androidAppVersion` so it's easier to disambiguate in plausible?

Reply:
Done